### ISSUE TITLE
Pixelbook 2017 install successful,but intel hd 615 grapchic card lost

### ISSUE THREAD

**Issue by FMLIFE** [yusefnapora/pixelbook-linux#65]
Fisrt, thanks a lot for your work,it's a great job!!
I installed ubuntu 20.10 (dual-boot),and the Graphics was recognized as Intel HD 615 correctly.
But after install your scripts, everything works great(touchpad,audio,keyboard backlight,screen brightness) except the graphic card, which was recognized as "llvmpipe (LLVM 11.0.0, 256 bits) / llvmpipe (LLVM 11.0.0, 256 bits)"
Does it related to the kernal?
what should i do?
thanks 
![Screenshot from 2020-11-20 03-47-59](https://user-images.githubusercontent.com/3346586/99779571-3c260700-2ae3-11eb-95ce-b229ba3a275d.png)

**Comment by raybi-asus** [comment]
Hi 

Do you mean the script runs well under 20.10?  I tried to install the script in 20.04, Then I got a black screen ...
Does It run well in 20.10??

**Comment by FMLIFE** [comment]
> Hi
> 
> Do you mean the script runs well under 20.10? I tried to install the script in 20.04, Then I got a black screen ...
> Does It run well in 20.10??

yes, script runs well under 20.10, and 20.10 still works(except the audio)now. I also encounter the black screen issue(under both 20.04 and 20.10), and solved by reinstall gnome-desktop.
1.press CTRL + ALT + back arrow(next to the ESC), to open tty terminal
2.setup network to connect to the internet(google netplan)
3.install gnome-deskop environment
4.run startx command

**Comment by raybi-asus** [comment]
Thanks for your comments!! A huge help for me ..
I will try it today ... Actually as you know 1910 is not supportted now , software source is out of service.
Hence I didn't know what I should do until I got your response ..

Thanks A lot !!

**Comment by FMLIFE** [comment]
You are welcome
Maybe the main problem is that even if the script includes desktop parameter, but actually  it was not installed or something else, the GUI missing(when run startx, the command not found),  so install the GUI will help
when you get into the tty, there is a warning( like "unminimize"), you can follow the instruction to have a try.
glad to help

**Comment by ackstorm23** [comment]
the screen is black because of the chromium kernel which is installed by 
the playbook.

if you use the distro kernel your video will work fine, however the backlit 
control and audio will not function as those are dependant on kernel 
patches the distro kernel does not have.

**Comment by raybi-asus** [comment]
> s that even if the script includes desktop parameter, but actually it was n

Thanks for clarify me. 
Could you please privode me some suggestions ? 
Do we need to upgrade to the latest distro ? As you know 1910 is not supportted now , and the software source is out of date. If i do upgrade to 20.04LTS, probably we will get black screen. FMLIFE already provide us a workaround for this issue , do you have any other idea? 

Thanks a lot in advance.

**Comment by soppelmann** [comment]
You need to downgrade mesa
